### ISSUE TITLE
Allow comma separated entries on a single line in outline object/array literals and outline arguments.

### ISSUE THREAD

**Issue by KodyJKing** [glassorg/ionscript#1]
For example,
```
let navState = {}
    grabX, setGrabX
    viewX, setViewX
    viewWidth, setViewWidth
let identity = []
    1, 0, 0
    0, 1, 0
    0, 0, 1
let foo = clamp()
    min, max
    x
```

**Comment by krisnye** [comment]
You know, if you put the {} or [] on a new line, that works today.
```
let navState =
    {}
        grabX, setGrabX
        viewX, setViewX
        viewWidth, setViewWidth
let identity =
    []
        1, 0, 0
        0, 1, 0
        0, 0, 1
let foo = clamp()
    min, max
    x
```

**Comment by krisnye** [comment]
This has been fixed and published as 0.5.0
